### PR TITLE
Added Apple's privacy declaration doc

### PR DESCRIPTION
**Pull request by DH-555**
Since App Store requires it starting may 1st, added the needed PrivacyInfo.xcprivacy file with it's content.

Related issue: https://github.com/langx/langx/issues/666

### CODE REVIEW

File: ios/App/App.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -16,6 +16,8 @@
 		504EC3121FED79650016851F /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 504EC3101FED79650016851F /* LaunchScreen.storyboard */; };
 		50B271D11FEDC1A000F3C39B /* public in Resources */ = {isa = PBXBuildFile; fileRef = 50B271D01FEDC1A000F3C39B /* public */; };
 		76D791412AE18684001083C5 /* GoogleService-Info.plist in Resources */ = {isa = PBXBuildFile; fileRef = 76D791402AE18684001083C5 /* GoogleService-Info.plist */; };
+		2EA9F22A2BD4347F00CAE2F3 /* PrivacyInfo.xcprivacy in Resources */ = {isa = PBXBuildFile; fileRef = 2EA9F2292BD4347F00CAE2F3 /* PrivacyInfo.xcprivacy */; };
```
Comment (xuelink) [inline ios/App/App.xcodeproj/project.pbxproj:19]: Did you build it ? :D

Comment (DH-555) [reply ios/App/App.xcodeproj/project.pbxproj:19]: Only tried in my app

Comment (DH-555) [reply ios/App/App.xcodeproj/project.pbxproj:19]: It's generated by XCode automatically when you create the file

**Review by xuelink — APPROVED** [review]
Works fine with simulator, no issues.

**Comment by xuelink** [comment]
Thank you for this PR, @DH-555 , appriciated !!

**Pull request merged**